### ISSUE TITLE
Error creating NS when joining

### ISSUE THREAD

**Issue by sirjofri** [9mine/9mine#73]
When joining the user gets kicked. The message from the server is:

```
Access denied. Reason: Kicked: Error creating NS. Try again later. Log:
/usr/local/share/lua/5.1/9p.lua:378: file '/n/9mine/user' not found
```

**Comment by metacoma** [comment]
Minetest server log for this session:
```
2020-12-10 12:34:53: ACTION[Server]: sirjofri [10.115.130.1] joins game. List of players: sirjofri
Write ndb/regquery -n user sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
Write mkdir /n/sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
Write mount -A  /n/sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
Write ndb/regquery -n user sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
Write mount -A  /n/sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
Write ndb/regquery -n user sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
Write mount -A  /n/sirjofri> /tmp/cmdchan_output >[2=1] to /tmp/file2chan/cmd
```

![screencast](https://user-images.githubusercontent.com/5146707/101774202-38483c00-3b07-11eb-98ed-0f35fe5e95d9.png)

**Comment by metacoma** [comment]
a restart of the minetest server and inferno helped

 I will leave this bug open because it seems there are some connection issues between minetest and inferno

**Comment by metacoma** [comment]
Fixes by #81
